src/shared/layouts/BaseLayout/components/TopBar/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -36,7 +36,7 @@ export const Breadcrumbs = () => {
         ) : (
           <StyledPlaceholder>
             <StyledLabelSmall color={variables.palette.on_surface}>
-              {label.substring(0, 1).toUpperCase()}
+              {label?.substring(0, 1).toUpperCase()}
```

Comment:
maybe add || '' or smth like this to not show undefined?
or it's not possible that label will be undefined?

Reply:
fixed